seacrowd/sea_datasets/id_sentiment_analysis/id_sentiment_analysis.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = """\
+@misc{ridife2019idsa,
+  author = {Fe, Ridi},
+  title = {Indonesia Sentiment Analysis Dataset},
+  year = {2019},
+  publisher = {GitHub},
+  journal = {GitHub repository},
+  howpublished = {\\url{https://github.com/ridife/dataset-idsa}}
+}
+"""
+
+_DATASETNAME = "id_sentiment_analysis"
+
+_DESCRIPTION = """\
+This dataset consists of 10806 labeled Indonesian tweets with their corresponding sentiment analysis: positive, negative, and neutral, up to 2019.
+This dataset was developed in Cloud Experience Research Group, Gadjah Mada University.
+There is no further explanation of the dataset. Contributor found this dataset after skimming through "Sentiment analysis of Indonesian datasets based on a hybrid deep-learning strategy" (Lin CH and Nuha U, 2023).
+"""
+
+_HOMEPAGE = "https://github.com/ridife/dataset-idsa/blob/master/Indonesian%20Sentiment%20Twitter%20Dataset%20Labeled.csv"
```

Comment:
this is more suitable to be placed in `_HOMEPAGE` rather than the github URL dataset:
https://ridi.staff.ugm.ac.id/2019/03/06/indonesia-sentiment-analysis-dataset/